mjpc/tasks/humanoid/skateboard/steering.cc

```diff
@@ -282,24 +299,26 @@ std::vector<double> ComputeTrackingResidual(const mjModel *model, const mjData *
 
     // current frame
     // mju_copy( result, mocap_translated + 3 * body_mocapid, 3);
-    mju_scl3(
-        result,
-        mocap_translated.data() + (model->nmocap - 1) * 3 * key_index_0 + 3 * body_mocapid,
```

Comment:
This index seems incorrect to me. The error remains in my change too.

Same below.